### PR TITLE
Make github avatar used configurable instead of hard-coded

### PR DESCRIPTION
**Pull request by deargle**
conflict resolved from justinaiken/retlab PR

-----
[View rendered README.md](https://github.com/deargle/retlab/blob/patch-github-avatar/README.md)

### CODE REVIEW

**Comment by deargle** [comment]
replaces / closes #3

File: _includes/mini-bio.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   {% assign bio = bio[0].content | newline_to_br | split:"<br />" %}
   {% assign about_url = "/about/" | relative_url %}
   <div class="col-sm-2">
-    <div class="float-sm-right pt-1">{% avatar benbalter size=100 %}</div>
+    <div class="float-sm-right pt-1">{% avatar {{site.github_username}} size=100 %}</div>
```
Comment (benbalter) [inline _includes/mini-bio.html]: We could also user `site.github.owner_name` to avoid hardcoding the username entirely.

Comment (deargle) [reply _includes/mini-bio.html]: I was about to raise the concern that that would tie the theme to github-pages-hosted builds only, but the theme already has a few other dependencies on `site.github`, so no harm in one more -- I like it.

**Comment by JustinAiken** [comment]
Thanks both of you! :)

**Pull request merged**